packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -189,6 +189,12 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
         return;
       }
 
+      // State is going from connected -> disconnected, emit the callEnded event.
+      const activeCall = this._call;
+      if (activeCall && activeCall.state === 'Connected' && clientState.calls[activeCall.id].state === 'Disconnected') {
```

Comment:
not final code in any way, this would instead use the same function that is used to check for end call page and do:
```
if (getPage(currentState) !== 'endcallpage' && getPage(newState) === 'endcallpage')
```